addons/website_sale_slides/models/slide_channel.py

```diff
@@ -46,7 +46,6 @@ def write(self, vals):
 
     def _synchronize_product_publish(self):
         self.filtered(lambda channel: channel.is_published and not channel.product_id.is_published).sudo().product_id.write({'is_published': True})
-        self.filtered(lambda channel: not channel.is_published and channel.product_id.is_published).sudo().product_id.write({'is_published': False})
```

Comment:
Also, if new courses are unpublished by default (at least true in 13.0), maybe we can also drop L37-L38, and therefore the `create` method ?

Reply:
By default and for pretty much all records that can be published, the value is set to False. But it's still possible to pass the values manually by calling the `create` method. So for each record where the `is_published` was set to True, the product should also be synchronized.

Reply:
Oh, right. Then, couldn't it also be part of `create`'s logic for calling `_synchronize_product_publish` ?

Reply:
On line 37, I mean 
Why not use this as in master:         
```python
channels.filtered(lambda channel: channel.enroll == 'payment')._synchronize_product_publish()
```
and add
```python
channels.filtered(lambda channel: channel.enroll == 'payment' and channel.is_published)._synchronize_product_publish()
```

Reply:
Thanks for the live answer